resources/dicts/patrols/plains/training/any.json

```diff
@@ -2828,5 +2828,125 @@
                 "lost_cats": ["r_c"]
             }
         ]
+    },
+    {
+        "patrol_id": "gen_train_amentorswisdom",
+        "biome": ["Any"],
+        "season": ["Any"],
+        "types": ["training"],
+        "tags": ["platonic","trust","respect","dislike","comfort"],
+        "patrol_art": "train_general_intro",
+        "min_cats": 2,
+        "max_cats": 2,
+        "min_max_status": {
+            "normal adult": [0, 1],
+            "all apprentices": [0, 1]
+        },
+    
+        "weight": 20,
+        "chance_of_success": 50,
+        "intro_text": "p_l is surprised when app1 makes a mistake on a simple move p_l thought {PRONOUN/app1/subject} had mastered. Before p_l can say anything, app1 wails and runs out of the training grounds. When p_l finds {PRONOUN/app1/poss} huddled in camp, {PRONOUN/app1/subject} {VERB/app1/sob/sobs} out a confession: {PRONOUN/app1/subject} will never be ready to be a warrior.",
+        "decline_text": "It wouldn't help to talk to app1 while {PRONOUN/app1/subject} {VERB/app1/is/are} in this state. p_l tells {PRONOUN/app1/poss} {PRONOUN/p_l/subject}'ll leave {PRONOUN/app1/poss} alone, and to come find {PRONOUN/p_l/poss} if {PRONOUN/app1/subject}'d like to keep training today. app1 never does.",
+        "success_outcomes": [
+            {
+                "text": "p_l waits patiently for app1 to calm down. Once {PRONOUN/app1/subject} can listen, p_l tells {PRONOUN/app1/poss} a secret: no warrior has ever felt like a warrior. Not even p_l. You will always feel like you, p_l says, but don't be afraid. You <i>must</i> be you before you are a warrior. That is what c_n truly needs. You. ",
+                "exp": 10,
+                "stat_trait": ["loving","compassionate","wise","calm","thoughtful","faithful","loyal","sincere","righteous"],
```

Comment:
I adore this text but you're missing a field here, the "can_have_stat": ["p_l"], which will tell the code that only p_l can trigger this stat outcome, not app1. You also need to refer to p_l as s_c here rather than p_l. This is because it makes finding typos in teh code easier and keeps things consistent :)